examples/popmon_dp_loader_example.ipynb

```diff
@@ -0,0 +1,479 @@
+    "    - Dataprofiler's examples using both dataloaders\n",
+    "    - Usage of the pm_stability_report function (popmon reports)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aec2198a",
+   "metadata": {},
+   "source": [
+    "## How to Install Popmon\n",
+    "To install popmon you can use the command below:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4383ed2a",
+   "metadata": {},
+   "source": [
+    "`pip3 install popmon`\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91dedc34",
+   "metadata": {},
+   "source": [
+    "From here, we can import the libararies needed for this demo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2adec556",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "try:\n",
+    "    sys.path.insert(0, '..')\n",
+    "    import dataprofiler as dp\n",
+    "except ImportError:\n",
+    "    import dataprofiler as dp\n",
+    "import pandas as pd\n",
+    "import popmon  # noqa"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ed532ec",
+   "metadata": {},
+   "source": [
+    "## Comparison of Dataloaders"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cccbf4cd",
+   "metadata": {},
+   "source": [
+    "First, we have the original pandas dataloading which works for specific file types. \n",
+    "This is good for if the data format is known ahead of time but is less useful for more dynamic cases."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96e9ff89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def pm_dataloader(path, time_index):\n",
+    "    # Load pm dataframe (Can only read csvs unless reader option is changed)\n",
+    "    if not time_index is None:\n",
+    "        pm_data = pd.read_csv(path, parse_dates=[time_index])\n",
+    "    else:\n",
+    "        time_index = True\n",
+    "        pm_data = pd.read_csv(path)\n",
+    "    return pm_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16dfbe10",
+   "metadata": {},
+   "source": [
+    "Next, we have the dataprofiler's dataloader. This allows for the dynamic loading of different data formats which is super useful when the data format is not know ahead of time.\n",
+    "This is intended to be an improvement on the dataloader standardly used in pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07481259",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def dp_dataloader(path, time_index):\n",
+    "    # Datalaoder from dataprofiler used\n",
+    "    dp_data = dp.Data(path) \n",
+    "\n",
+    "    # Make dataframe from dataprofiler popmon compliant\n",
+    "    if not time_index is None:\n",
+    "        # Mimics parse_dates=[time_index] optional param\n",
+    "        dp_data.data[time_index] = pd.to_datetime(dp_data.data[time_index])\n",
```

Comment:
instead of passing in time_index. we could use the profiler capabilities to check if any column is datetime automatically.

e.g. use the `datetime_column` profiler
```python
from dataprofiler.profilers.datetime_column_profile import DateTimeColumn

for col in ...:
    datetime_profile = DateTimeColumn(name='check_col')
    datetime_profile.update(dp_data.iloc[col].apply(str))  # unsure if iloc correct off the top of my head
    if datetime_profile.data_type_ratio == 1:    # perfect match, above doesn't remove nulls, so may have issue with nulls, would need to solve
        dp_data.data.iloc[col] = pd.to_datetime(dp_data.data.iloc[col])
```

Reply:
could also use DPs cleaner funcs too prior to sending in as well to remove nulls